build-ps/percona-server-8.0_builder.sh

```diff
@@ -670,6 +670,10 @@ build_deb(){
     fi
 
     if [ ${DEBIAN_VERSION} = "artful" -o ${DEBIAN_VERSION} = "bionic" -o ${DEBIAN_VERSION} = "cosmic" ]; then
```

Comment:
Shouldn't you remove "cosmic" from here ?

Reply:
fixed